test/fixtures/labels.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
You are not using this file. Remove it and in future don't add any piece of code that is not used.